uint64 timestamp			# time since system start (microseconds)
uint8 state        		# See enum COMPANION_PROCESS_STATE
uint8 component         # See enum MAV_COMPONENT

uint8 MAV_STATE_UNINIT = 0
uint8 MAV_STATE_BOOT = 1
uint8 MAV_STATE_CALIBRATING = 2
uint8 MAV_STATE_STANDBY = 3
uint8 MAV_STATE_ACTIVE = 4
uint8 MAV_STATE_CRITICAL = 5
uint8 MAV_STATE_EMERGENCY = 6
uint8 MAV_STATE_POWEROFF = 7
uint8 MAV_STATE_FLIGHT_TERMINATION = 8

uint8 MAV_COMP_ID_ALL = 0
uint8 MAV_COMP_ID_AUTOPILOT1 = 1
uint8 MAV_COMP_ID_CAMERA = 100
uint8 MAV_COMP_ID_CAMERA2 = 101
uint8 MAV_COMP_ID_CAMERA3 = 102
uint8 MAV_COMP_ID_CAMERA4 = 103
uint8 MAV_COMP_ID_CAMERA5 = 104
uint8 MAV_COMP_ID_CAMERA6 = 105
uint8 MAV_COMP_ID_SERVO1 = 140
uint8 MAV_COMP_ID_SERVO2 = 141
uint8 MAV_COMP_ID_SERVO3 = 142
uint8 MAV_COMP_ID_SERVO4 = 143
uint8 MAV_COMP_ID_SERVO5 = 144
uint8 MAV_COMP_ID_SERVO6 = 145
uint8 MAV_COMP_ID_SERVO7 = 146
uint8 MAV_COMP_ID_SERVO8 = 147
uint8 MAV_COMP_ID_SERVO9 = 148
uint8 MAV_COMP_ID_SERVO10 = 149
uint8 MAV_COMP_ID_SERVO11 = 150
uint8 MAV_COMP_ID_SERVO12 = 151
uint8 MAV_COMP_ID_SERVO13 = 152
uint8 MAV_COMP_ID_SERVO14 = 153
uint8 MAV_COMP_ID_GIMBAL = 154
uint8 MAV_COMP_ID_LOG = 155
uint8 MAV_COMP_ID_ADSB = 156
uint8 MAV_COMP_ID_OSD = 157
uint8 MAV_COMP_ID_PERIPHERAL = 158
uint8 MAV_COMP_ID_QX1_GIMBAL = 159
uint8 MAV_COMP_ID_FLARM = 160
uint8 MAV_COMP_ID_MAPPER = 180
uint8 MAV_COMP_ID_MISSIONPLANNER = 190
uint8 MAV_COMP_ID_PATHPLANNER = 195
uint8 MAV_COMP_ID_OBSTACLE_AVOIDANCE = 196
uint8 MAV_COMP_ID_VISUAL_INERTIAL_ODOMETRY = 197
uint8 MAV_COMP_ID_IMU = 200
uint8 MAV_COMP_ID_IMU_2 = 201
uint8 MAV_COMP_ID_IMU_3 = 202
uint8 MAV_COMP_ID_GPS = 220
uint8 MAV_COMP_ID_GPS2 = 221
uint8 MAV_COMP_ID_UDP_BRIDGE = 240
uint8 MAV_COMP_ID_UART_BRIDGE = 241
uint8 MAV_COMP_ID_SYSTEM_CONTROL = 250

